scripts/release.sh

```diff
@@ -43,16 +41,54 @@ fi
 develop_branch="develop"
 main_branch="master"
 
-# Stash any local changes to avoid errors during checkout
-git status
-prompt_user "Would you like to stash any local changes"
-if [[ $? == 1 ]]; then
-    # Stash local changes to prevent issues with checkout
-    git stash
-    echo "use \"git stash pop\" when this script is complete to restore your changes"
+
+
+# Checkout develop - so we can update versions.
+# We need to checkout the branch before we start modifying files.
+current_branch=$(git branch --show-current)
```

Comment:
![Screen Shot 2022-07-25 at 1 49 38 PM](https://user-images.githubusercontent.com/1767063/180841975-b2764aed-885c-4dd5-a763-30cb2170f903.png)

Reply:
That command works for me.  I can look for an alternative command that will work in more cases.